blocks/library/heading/test/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { query, getBlocks, unregisterBlock, setUnknownTypeHandler } from 'api';
+import { blockSettings } from '../';
+
+describe( 'heading block', () => {
+	afterEach( () => {
+		setUnknownTypeHandler( undefined );
+		getBlocks().forEach( ( block ) => {
+			unregisterBlock( block.slug );
+		} );
+	} );
+
+	it( 'should parse the block properly', () => {
+		const rawContent = '<h4 style="text-align: right;">Chicken</h4>';
+		const attrs = query.parse( rawContent, blockSettings.attributes );
```

Comment:
What of a block would we really care to test? Maybe we could export these individually rather than the entire block settings object?

```js
export const attributes = {
	// ...
};

registerBlock( 'core/heading', {
	attributes,

	// ...
} );
```